### ISSUE TITLE
What version should we use?

### ISSUE THREAD

**Issue by Ilyes512** [nodenv/nodenv#63]
I am trying to create an Ansible role to automate installing nodenv on Ubuntu, but I would like to know what version of nodenv to install?

So there is the `master` branch, which by default would be used if you fallow the install instructions. And then I also see tags (latest is `v1.0.0`). 

- [v1.0.0](https://github.com/nodenv/nodenv/commits/v1.0.0) is tagged on 27 Dec **2015**.
- Last commit on [master](https://github.com/nodenv/nodenv/commits/master) is on 21 Dec **2016**.

A total of (721 - 636 =) 85 commit's and roughly 1 year since the last tag.

So to summarise my questions:
- What are your recommendations for development and production environments?
- Can there be a new release (tag)?

**Comment by jasonkarns** [comment]
Released 1.1.0: https://github.com/nodenv/nodenv/releases/tag/v1.1.0
